chia/data_layer/data_layer_wallet.py

```diff
@@ -107,7 +107,8 @@ def from_json_dict(cls, json_dict: Dict[str, Any]) -> "Mirror":
 @final
 class DataLayerWallet:
     if TYPE_CHECKING:
-        _protocol_check: ClassVar[WalletProtocol] = cast("DataLayerWallet", None)
+        # TODO Create DataLayer coin data model if necessary
+        _protocol_check: ClassVar[WalletProtocol[Any]] = cast("DataLayerWallet", None)
```

Comment:
`object` or `None` instead of `Any` might be better.  Either one would keep us from doing stuff with the parameter whereas `Any` lets us pretend it is anything we want without any hinting errors.